pumpkin-protocol/src/server/config/s_cookie_response.rs

```diff
@@ -0,0 +1,14 @@
+use pumpkin_macros::server_packet;
+
+use crate::{Identifier, VarInt};
+
+#[derive(serde::Deserialize)]
+#[server_packet("config:cookie_response")]
+/// Response to a Cookie Request (configuration) from the server.
+/// The Notchian server only accepts responses of up to 5 kiB in size.

```

Comment:
```suggestion
/// The Notchian (vanilla) server only accepts responses of up to 5 kiB in size.
```
Same reason as previously.